demo/demo.ts

```diff
@@ -48,7 +47,7 @@ import {
            </div>
         </div>
         <div class="col-md-4">
-          <h3>{{ date | calendarTitle:view }}</h3>
+          <h3>{{ date | calendarDate:'dayViewTitle' }}</h3>
```

Comment:
got it. I am going to push the update now, however I am curious as to why this is the case? I didn't receive any error in the  console but I am genuinely interested as to why it works like that (if there's a quick explanation for it, of course 😉)